### PR TITLE
Improve scanner console output

### PR DESCRIPTION
**Pull request by sonninnos**
## Description

Just a tiny bit better readability to the console output by adding the target playlist, adding a space in the beginning, and coloring `Add` green. Are there cases where these won't work or are not enough? Windows coloring works both in native command prompt and in msys2 bash, and ANSI method works only in bash.

```
# ra-dev --scan s:\emu\amiga\lha\y
Scanning content: "s:\emu\amiga\lha\y"..
1/5: YogisBigCleanUp_v1.01_2171.lha..
 Add "Yogi's Big Clean Up (Europe)" to "Commodore - Amiga.lpl"
2/5: YogisGreatEscape_v1.1_1355.lha..
 Add "Yogi's Great Escape (Europe)" to "Commodore - Amiga.lpl"
3/5: YoJoe_v1.0_0750.lha..
 Add "Yo! Joe! (Europe)" to "Commodore - Amiga.lpl"
4/5: Yolanda_v1.0a_Files.lha..
 Add "Yolanda (Europe)" to "Commodore - Amiga.lpl"
5/5: YogiBear&Friends_v1.0.lha..
 Add "Yogi Bear & Friends in the Greed Monster (Europe)" to "Commodore - Amiga.lpl"
Scanning of directory finished
```

image

### CODE REVIEW

**Comment by hizzlekizzle** [comment]
eyyy, colored text, very nice. Is that something we should/could extend to, say, warnings [yellow] and errors [red]?

**Comment by sonninnos** [comment]
That sounds reasonable and doable, since it isn't that complicated after all.

**Comment by i30817** [comment]
You can also make hyperlinks and hover text in modern cmd line. I was surprised too.

**Comment by i30817** [comment]
A unrelated question the database for amiga has whdloads now?

**Comment by i30817** [comment]
The whdload set is changing all the time, considering it gets bugfixes almost every month for about 20 years now. I don't know how they do it, but it's kind of a problem to keep up-to-date if you have to do any kind of manual transformation.

File: tasks/task_database.c [hunk 1]
```diff
@@ -804,9 +804,26 @@ static int database_info_list_iterate_found_match(
...
+         {
+            SetConsoleTextAttribute(con, 2);
+            WriteConsole(con, " Add ", 5, NULL, NULL);
+            SetConsoleTextAttribute(con, 7);
+         }
+         else
+            fputs(" Add ", stdout);
+#else
+         fputs("\x1B[32m Add \x1B[0m", stdout);
+#endif
```
Comment (RobLoach) [inline tasks/task_database.c:823]: While I appreciate the colouring, I think it's adding code complexity over a simple `RARCH_LOG()`.

Comment (sonninnos) [reply tasks/task_database.c:823]: Hmm, this console output is separate from actual logging (which is also done), and colors are useless when logging to a file, so not sure how complex things ought to get..

Comment (RobLoach) [reply tasks/task_database.c:823]: Hmm, i see :thinking:

**Comment by sonninnos** [comment]
Current layout:
image

Additions are now green `++`, dupes yellow `==` and no match red `??`.

**Pull request merged**